MANIFEST.in

```diff
@@ -1,5 +1,5 @@
 include README.md
 include LICENSE.txt
-recursive-include src/ *.py
+recursive-include src/ufonormalizer/ *.py
```

Comment:
when you use setuptools_scm to track the version from git, it will also automatically include all the files in the git repository to the source distribution manifest. So in that case the MANIFEST.in is only useful if you wish to exclude some files that would normally be included because they are part of the git repository. In any case, there is no need to include *.py files (they are automatically added to the sdist manifest by setuptools as long as they are listed in setup.py's py_modules or packages keyword).

Reply:
Thanks for the advice. @benkiel has already merged this, but I'll make a note to clean this up next time I work on this repo.